app.js

```diff
@@ -92,7 +93,15 @@ app.use((err, req, res, next) => {
   logger.error(err, 'burped error');
   res.status(err.status || 500).json({msg: err.message});
 });
+
 const httpServer = app.listen(PORT);
+process.on('uncaughtException', function(err) {
+    if(err.errno === 'EADDRINUSE' && useMultiple) {
+        const httpServer = app.listen(PORT++);
```

Comment:
We should burp out a log message here that we are now going to attempt to bind to a new port

Reply:
added.